packages/docusaurus-plugin-content-blog/src/__tests__/__fixtures__/website/build-snap/blog/blog-with-links/index.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
```

Comment:
How was this file generated?

Reply:
I created the blog-with-links.mdx in the website/blog folder and ran yarn:build:website:blogOnly. If this is not the way how the files in __fixtures__ created, please let me know the create way to do it.